calico.go

```diff
@@ -251,28 +257,23 @@ func cmdAdd(args *skel.CmdArgs) error {
 			// Otherwise, incoming traffic is only allowed from profiles with the same tag.
 			fmt.Fprintf(os.Stderr, "Calico CNI creating profile: %s\n", conf.Name)
 			var inboundRules []api.Rule
-			if orchestrator == "k8s" {
+			if epIDs.Orchestrator == "k8s" {
 				inboundRules = []api.Rule{{Action: "allow"}}
 			} else {
 				inboundRules = []api.Rule{{Action: "allow", Source: api.EntityRule{Tag: conf.Name}}}
 			}
 
-			profile := &api.Profile{
-				Metadata: api.ProfileMetadata{
-					Name: conf.Name,
-					Tags: []string{conf.Name},
-				},
-				Spec: api.ProfileSpec{
-					EgressRules:  []api.Rule{{Action: "allow"}},
-					IngressRules: inboundRules,
-				},
-			}
+			profile := api.NewProfile()
```

Comment:
not sure why this changed.  The old code was more readable.  

I think it's because you thought we needed the TypeMetadata, but we shouldn't.

Reply:
ok if it works without metadata then I'll use the old style of code